fix: re-write the web role to be "full stack web" (#345)
shorebirdtech/website: src/content/jobs/web-software-engineer.md

### What You'll Do

> - Own our front-ends. This includes our shorebird.dev, console.shorebird.dev,

Reviewer: ```suggestion
- Own our front-ends. This includes shorebird.dev, console.shorebird.dev,
```

  admin.shorebird.dev (private), and other projects.

- Implement web-facing features end-to-end. We use small "backends for frontend"

>   written in React, talking through to other apis or (in a few cases) our
>   Postgres database directly with prisma.

Reviewer: ```suggestion
  written in React, which talk to our backend APIs or (in a few cases) our
  Postgres database directly with prisma.
```
